src/blueapi/service/app.py

```diff
@@ -29,7 +27,7 @@ class Service:
     def __init__(self, config: ApplicationConfig) -> None:
         self._config = config
         self._ctx = BlueskyContext()
-        self._ctx.with_startup_script(self._config.env.startup_script)
+        self._ctx.with_startup_script(config.env.startup_script)
```

Comment:
Can revert this line as not making any functional change and preserves the git blame